fastparquet/converted_types.py

```diff
@@ -91,15 +91,15 @@ def convert(data, se):
         scale_factor = 10**-se.scale
         return data * scale_factor
     elif ctype == parquet_thrift.ConvertedType.DATE:
-        return pd.to_datetime(data.map(datetime.date.fromordinal), box=False)
+        return pd.to_datetime(data.map(datetime.date.fromordinal))
     elif ctype == parquet_thrift.ConvertedType.TIME_MILLIS:
-        return pd.to_timedelta(data, unit='ms', box=False)
+        return pd.to_timedelta(data, unit='ms')
     elif ctype == parquet_thrift.ConvertedType.TIMESTAMP_MILLIS:
-        return pd.to_datetime(data, unit='ms', box=False)
+        return pd.to_datetime(data, unit='ms')
     elif ctype == parquet_thrift.ConvertedType.TIME_MICROS:
-        return pd.to_timedelta(data, unit='us', box=False)
+        return pd.to_timedelta(data, unit='us')
     elif ctype == parquet_thrift.ConvertedType.TIMESTAMP_MICROS:
-        return pd.to_datetime(data, unit='us', box=False)
+        return pd.to_datetime(data, unit='us')
```

Comment:
We do know that in the function above, read_col. 
Actually just checked, and the special NaT value (`pd.to_datetime('NaT').value`) can be passed totally fine.